### PR TITLE
Update installation page.

### PR DESCRIPTION
**Pull request by luketheobscure**
- Fix learn-es2015 link
- Add note about downloading node directly
- Remove PhantomJS section

### CODE REVIEW

File: source/getting-started/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This tool provides:
 * Modern application asset management (including concatenation, minification, and versioning).
 * Generators to help create components, routes, and more.
 * A conventional project layout, making existing Ember applications easy to approach.
-* Support for ES2015/ES6 JavaScript via the [Babel](http://babeljs.io/docs/learn-es2015/) project. This includes support for [JavaScript modules](http://exploringjs.com/es6/ch_modules.html), which are used throughout this guide.
+* Support for ES2015/ES6 JavaScript via the [Babel](https://babeljs.io/learn-es2015/) project. This includes support for [JavaScript modules](http://exploringjs.com/es6/ch_modules.html), which are used throughout this guide.
```
Comment (luketheobscure) [inline source/getting-started/index.md]: Current link just redirects to the new one.

File: source/getting-started/index.md [hunk 2]
```diff
@@ -35,7 +35,8 @@ If you get a *"command not found"* error or an outdated version for Node:
 
 * Windows or Mac users can download and run [this Node.js installer](http://nodejs.org/en/download/).
 * Mac users often prefer to install Node using [Homebrew](http://brew.sh/). After
-installing Homebrew, run `brew install node` to install Node.js.
+installing Homebrew, run `brew install node` to install Node.js. Alternatively, installer packages are available directly
+from [Node.js](https://nodejs.org/en/download/).
```
Comment (luketheobscure) [inline source/getting-started/index.md]: Lately I've been recommending to new developers to install it this way. For some reason I've helped a half-dozen people fix where they've somehow installed it with `sudo` and then can't install global npm packages without elevated privileges. The installers mitigate that.

File: source/getting-started/index.md [hunk 3]
```diff
@@ -44,11 +45,6 @@ If you get an outdated version of npm, run `npm install -g npm`.
 
 On Mac and Linux, you can improve file watching performance by installing [Watchman](https://facebook.github.io/watchman/docs/install.html).
 
-### PhantomJS (optional)
-
-You can run your tests from the command line with PhantomJS, without the
-need for a browser to be open. Consult the [PhantomJS download instructions](http://phantomjs.org/download.html).
-
```
Comment (luketheobscure) [inline source/getting-started/index.md]: I'm not sure how you handle the guides being updated as versions are released, but I think this is slated to be replaced in v2.15 ([relevant commit](https://github.com/ember-cli/ember-cli/commit/2001c2d93359c42b14dc8da9775948d04bd6e188#diff-f7e2d0d5944207cb98b51991f8da57b0)).

Comment (toddjordan) [inline source/getting-started/index.md]: 👍 We'll drop this when 2.15 comes out in the next few days.

**Comment by toddjordan** [comment]
Thanks ❤️ 

Will merge when 2.15 ships

**Pull request merged**